src/setup/i18n.ts

```diff
@@ -8,13 +8,22 @@ export const I18N_FALLBACK = 'de';
 export const I18N_FALLBACK_MESSAGES = fallbackMessages;
 export const I18N_LOCALES = [I18N_FALLBACK, 'fr'];
 
+const datetimeFormats: IntlDateTimeFormat = {
+  month: {
+    month: 'long',
+  },
+  weekday: { // Monday, Tuesday, Wednesday, ...
+    weekday: 'long',
+  },
+};
+
 // Add styleguide only translations
 if (process.env.NODE_ENV !== 'production') {
-  const styleguideTranslations = require('./styleguide.translations.json'); // eslint-disable-line global-require
+  const styleguideTranslations = require('./styleguide.translations.json'); // eslint-disable-line global-require, @typescript-eslint/no-var-requires
```

Comment:
suggestion: I would recommend we disable this rule in the configuration package all together, since we only use `require` in the styleguide context anyway, so there should be no issue for the productive context anyway. What is your thought?

Reply:
Yes, fine with me.